### PR TITLE
3getapi

### PR DESCRIPTION
**Pull request by Asileka**
3. GET /api/categories
https://trello.com/b/SgkPhIcH/kats-ncgames-board

### CODE REVIEW

File: __tests__/ncgamesapp.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+afterAll(() => {
+  return connection.end();
+});
+describe("GET:/api/categories", () => {
+  it("GET api/categories responds with categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.categories).toHaveLength(4);
```
Comment (douglashellowell) [inline __tests__/ncgamesapp.test.js:18]: Your tests here are only checking length of the array

You should test the contents are correct too!

File: __tests__/ncgamesapp.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      });
+  });
+  it("GET api/badroute respons with 204 error", () => {
+    return request(app).get("/api/badroute").expect(204);
```
Comment (douglashellowell) [inline __tests__/ncgamesapp.test.js]: `204` isn't an error code - i think you mean `404`?

**Comment by douglashellowell** [comment]
Nice work!

Your tests need a little bit more detail - see my other comments

**Comment by Asileka** [comment]
Thanks Doug, I'll fix those asap

**Review by Angaca — APPROVED** [review]
Ready to go now!

**Pull request merged**